### DISCUSSION TITLE
Should Network.http.pipelining.maxrequests and Network.http.pipelining be included

### DISCUSSION THREAD

**Discussion by Feejus** [yokoffing/Betterfox#282] [Ideas]
Hi there,

First, I would like to express my gratitude for the remarkable solution you have provided, which significantly enhances the speed and privacy of Firefox. I appreciate your dedication to improving Firefox's performance and overall user experience. 

However, after reviewing the code, I could not find the "Network.http.pipelining.maxrequests" and "Network.http.pipelining" settings. I was wondering if there is a specific technical reason or explanation behind their absence.

If there is no particular reason for excluding these settings, would you consider incorporating them in the upcoming release?

For your reference, I am currently running Firefox version 125b5, and it is working exceptionally well.

Thanx

**Comment by yokoffing** [comment]
https://github.com/yokoffing/Betterfox/issues/193

**Comment by Feejus** [comment]
Thanx, I did not know this. Looking online there are articles referring to this.
